internal/dag/dag.go

```diff
@@ -585,6 +585,18 @@ type SecureVirtualHost struct {
 	// only reason to set this to `true` is when you are migrating
 	// from internal to external authorization.
 	AuthorizationFailOpen bool
+
+	// AuthorizationBufferSettingsEnabled sets whether buffer settings are set for buffering request data sent to AuthorizationServer
+	AuthorizationBufferSettingsEnabled bool
```

Comment:
maybe wrap these fields in a struct and use a pointer to that struct in this object, can remove this boolean field with that

Reply:
Done